fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/populator/AbstractWorkbookPopulator.java

```diff
@@ -130,7 +130,7 @@ protected void setClientAndGroupDateLookupTable(Sheet sheet, List<ClientData> cl
                     }
                     if (containsClientExtId) {
                         if (client.getExternalId() != null) {
```

Comment:
This should be changed to `!client.getExternalId().isEmpty()`, no?

Reply:
Done